activerecord/test/cases/connection_adapters/connection_handler_test.rb

```diff
@@ -94,6 +94,8 @@ def test_establish_connection_using_3_level_config_defaults_to_default_env_prima
           ActiveRecord::Base.configurations = @prev_configs
           ENV["RAILS_ENV"] = previous_env
           ActiveRecord::Base.establish_connection(:arunit)
+          File.delete "db/primary.sqlite3"
+          Dir.delete "db"
```

Comment:
You could just use `FileUtils.rm_rf "db"` instead of the two commands

Reply:
Quite. Done.